eo-maven-plugin/src/main/java/org/eolang/maven/HashOfTag.java

```diff
@@ -56,19 +63,29 @@ final class HashOfTag {
      */
     public String hash() throws IOException {
         final String link = "https://home.objectionary.com/tags.txt";
-        final InputStream ins = new URL(link).openStream();
-        final Scanner scanner = new Scanner(ins);
-        while (scanner.hasNextLine()) {
-            final String line = scanner.nextLine();
-            final String[] parts = line.split("\t");
-            if (Objects.equals(parts[1], this.tag)) {
-                Logger.info(this, "Git sha of %s is %s", this.tag, parts[0]);
-                return parts[0];
+        String result = "";
+        if (REQUESTED_HASHES.containsKey(this.tag)) {
```

Comment:
@EugeneDar This is still not entirely thread-safe. 
I believe in this case the whole `hash()` method can be marked `synchronised`.

Reply:
@mximp I fixed it.